cg/services/post_processing/pacbio/data_transfer_service/data_transfer_service.py

```diff
@@ -0,0 +1,34 @@
+from cg.services.post_processing.abstract_classes import PostProcessingDataTransferService
+from cg.services.post_processing.pacbio.data_transfer_service.dto import (
+    PacBioDTOs,
+    PacBioSampleSequencingMetricsDTO,
+    PacBioSequencingRunDTO,
+    PacBioSMRTCellDTO,
+)
+from cg.services.post_processing.pacbio.data_transfer_service.utils import (
+    get_sample_sequencing_metrics_dtos,
+    get_sequencing_run_dto,
+    get_smrt_cell_dto,
+)
+from cg.services.post_processing.pacbio.metrics_parser.metrics_parser import PacBioMetricsParser
+from cg.services.post_processing.pacbio.metrics_parser.models import PacBioMetrics
+from cg.services.post_processing.pacbio.run_data_generator.run_data import PacBioRunData
+
+
+class PacBioDataTransferService(PostProcessingDataTransferService):
+
+    def __init__(self, metrics_service: PacBioMetricsParser):
+        super().__init__(metrics_service=metrics_service)
+
+    def get_post_processing_dtos(self, run_data: PacBioRunData) -> PacBioDTOs:
+        metrics: PacBioMetrics = self.metrics_service.parse_metrics(run_data)
```

Comment:
I'm guessing exceptions can be thrown from the metrics parsing, should they be caught here or just bubble up?

Reply:
we are fixing error handling in a later step